### PR TITLE
Fix file upload test and value validation

### PR DESCRIPTION
**Pull request by anandgavai**

### CODE REVIEW

File: app/scripts/app.js [hunk 1]
```diff
@@ -46,4 +46,5 @@ app.config(function ($routeProvider) {
       redirectTo: '/'
     });
 })
-  .constant('defaultFolder', '2016-01-14');
+  .constant('defaultFolder', '2016-01-14')
```
Comment (mkuzak) [inline app/scripts/app.js:49]: this is not a constant, it should be read from releases file

Comment (mkuzak) [inline app/scripts/app.js:49]: how is the default normative dataset connected with overlapping calendar?

**Comment by mkuzak** [comment]
https://github.com/andi-nl/ANDI-frontend/pull/86/commits/6148c64d8d9e18b34e3a6d11f71ec75ad98f6104 is mostly white space changes, please don't mix white space changes with logical changes. It also looks like the changes are from spaces to tabs which should not be used here as stated in `.editorconfig`. Please check your editorconfig configuration.

File: app/scripts/app.js (hunk 1 above)
Comment (anandgavai) [inline app/scripts/app.js:49]: Hi Mateusz it is not, it is one of the items that I fixed before sending pull request

Comment (mkuzak) [inline app/scripts/app.js:49]: _It is not constant_ or _it should not be constant_. We have discussed this and decided that default normative dataset should not be hardcoded (i.e constant) but should be set in release.json file.
Also this change does not mach the commit message and should not be here.

**Pull request closed without merging**